browser/help/topics/v4-hts.md

```diff
@@ -66,11 +66,336 @@ The above example will retrieve the entire frequency struct for each variant. To
+    - `AN`: Total number of alleles in the group with the maximum allele frequency.
+    - `homozygote_count`: Count of homozygous individuals in the group with the maximum allele frequency.
+    - `gen_anc`: Genetic ancestry group with maximum allele frequency
+  - `non_ukb`: grpmax information across the non-UKB subset of the gnomAD exomes.
+    - `AC`: Alternate allele count in the group with the maximum allele frequency.
+    - `AF`: Maximum alternate allele frequency, (AC/AN), across groups in gnomAD.
+    - `AN`: Total number of alleles in the group with the maximum allele frequency.
+    - `homozygote_count`: Count of homozygous individuals in the group with the maximum allele frequency.
+    - `gen_anc`: Genetic ancestry group with maximum allele frequency
+- `joint_freq`: Array of combined exomes and genomes allele frequency information (AC, AN, AF, homozygote count) across the full gnomAD release and for each genetic ancestry group. Note that the values in array will correspond to combined or joint value if the variant was present in both data types, otherwise this array will contain frequencies only for the data type associated with the Hail Table (in this case, exomes).
+    - AC: Combined (exomes + genomes) alternate allele count in release.
+    - AF: Combined (exomes + genomes) alternate allele frequency, (AC/AN), in release.
+    - AN: Total number of alleles across exomes and genomes in release.
+    - homozygote_count: Count of homozygous alternate individuals across exomes and genomes in release.
+- `joint_faf`: Array of combined exomes and genomes filtering allele frequency information (AC, AN, AF, homozygote count). Note that the values in array will correspond to the joint or combined value if the variant had a defined filtering allele frequency in both data types, otherwise this array will contain filtering allele frequencies only for the data type associated with the Hail Table (in this case, exomes).
+    - `faf95`: Combined exomes and genomes filtering allele frequency (using Poisson 95% CI).
+    - `faf99`: Combined exomes and genomes filtering allele frequency (using Poisson 99% CI).
+- `joint_grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency across both exomes and genomes. Excludes Amish (ami), Ashkenazi Jewish (asj), European Finnish (fin), and "Remaining individuals" (remaining) groups.
+    - `AC`: Alternate allele count in the group with the maximum allele frequency.
+    - `AF`: Maximum alternate allele frequency, (AC/AN), across groups in gnomAD.
+    - `AN`: Total number of alleles in the group with the maximum allele frequency.
+    - `homozygote_count`: Count of homozygous individuals in the group with the maximum allele frequency.
+    - `gen_anc`: Genetic ancestry group with maximum allele frequency.
+- `contingency_table_test`
+- `cochran_mantel_haenszel_test`
```

Comment:
Needs description for these two. How about: 
```suggestion
- `contingency_table_test`: Hail's `contingency_table_test` on the REF and ALT alleles counts between exomes and genomes. 
- `cochran_mantel_haenszel_test`: Cochran–Mantel–Haenszel test on the REF and ALT alleles counts between exomes and genomes using genetic ancestry group as the stratification.
```

Reply:
works for me -- I was hoping for @jkgoodrich's help with these (so will leave this open for now)